tests/page-objects/analytics/analytics.wdio.page.js

```diff
@@ -0,0 +1,45 @@
+const listModule = module => $(`.mm-navigation-menu li a[href="${module}"]`);
```

Comment:
There seems to be a lot of unused code in this PR. Can you remove what's not used?